acceptance/setup/packages/pre-suite/010_Install.rb

```diff
@@ -46,4 +46,8 @@
 }
 
 install_packages_on(master, MASTER_PACKAGES)
-install_packages_on(agents, AGENT_PACKAGES)
+if ENV['AIO_AGENT_INSTALL'].nil?
```

Comment:
Not a functional change, but for readability i'd reverse the if/else clauses to drop the nil?. i.e.

```
if ENV['AIO_AGENT_INSTALL']
  install_aio_on(agents)
else
  install_packages_on(agents, AGENT_PACKAGES)
end
```

One less negative to process and the "if aio, do aio stuff" reads cleanly.